ui/src/components/Header/Controller.tsx

```diff
@@ -72,22 +72,23 @@ export const Controller = (): ReactElement => {
                 }
               </Select>
             </FormControl>
-            <Button
-              variant="contained"
-              onClick={start}
-              endIcon={<PlayArrow />}>
-              Start
-            </Button>
-          </>
+            <Box>
+              <Button
+                variant="contained"
+                onClick={start}
+                startIcon={<PlayArrow />}>
+                Start
+              </Button>
+            </Box>
+          </Box>
         }
       </ButtonGroup>
       <Chip
-        style={{ borderRadius: 20 }}
         label={isRunning ? 'Running' : 'Stopped'}
-        color={isRunning ? 'success' : 'error'}
+        color={isRunning ? 'success' : 'warning'}
 
```

Comment:
```suggestion
sx={{ p: 2, borderRadius: 10 }}
```
Maybe like this looks a bit better? @lucbpz

Reply:
According to our [Design Extensions Styleguide](https://www.figma.com/file/U7pLWfEf6IQKUHLhdateBI/Docker-Design-Guidelines?node-id=1%3A28771) (you can find it in our [docs](https://docs.docker.com/desktop/extensions-sdk/design/design-guidelines/)) our Chips don't have that padding and border radius. This is because we try to provide out of the box components that match with each other, and whenever we make changes to our DS, all components are taken into consideration - if you use a `Chip` next to a `Typography` without custom styling, you can expect they would always look ok no matter if we update the theme.

But! With that said, this is our recommendation on how to use MUI components to match our DS. You can choose to style components according to your needs or to also align with your Designs. I'll leave the decision to commit the suggestion to you 😄